repository_service_tuf/cli/artifact/repository.py

```diff
@@ -0,0 +1,39 @@
+# SPDX-FileCopyrightText: 2023 VMware Inc
```

Comment:
Preferred to not mess a repository logic into the download client. 

Do you think it fits together with the upcoming `rstuf artifact repository` command?
It's again a client, so having the hash bin logic here may be confusing as well.

Reply:
I really don't understand this repository here :)